docs/source/features/tensorbay_client.rst

```diff
@@ -2,71 +2,65 @@
 
 ****************
  Dataset Client
 ****************
 
-:class:`Dataset Client <~graviti.client.dataset.DatasetClient>` deals with the operations
-inside a :ref:`basic_concepts:Dataset`.
-See :numref:`Table. %s <dataset_client>` for more details.
+**dataset client** deals with the operations inside a dataset.
+See :numref:`Table. %s <dataset_client_table>` for more details.
 
-.. _dataset_client:
+.. _dataset_client_table:
 
 .. table:: Dataset Client
    :align: center
    :width: 70%
 
    ==========================================  ================================================================
-   Ojects to be operated                       methods
+   Objects to be Operated                       Methods
    ==========================================  ================================================================
-   :ref:`basic_concepts:Segment`               :meth:`~graviti.client.dataset.DatasetClient.list_segment_names`
-                                               :meth:`~graviti.client.dataset.DatasetClient.get_segment_object`
-                                               :meth:`~graviti.client.dataset.DatasetClient.delete_segments`
-   :ref:`basic_concepts:Catalog & SubCatalog`  :meth:`~graviti.client.dataset.DatasetClient.get_catalog`
-                                               :meth:`~graviti.client.dataset.DatasetClient.upload_catalog`
-   :ref:`basic_concepts:Dataset`               :meth:`~graviti.client.dataset.DatasetClient.commit`
-                                               :meth:`~graviti.client.dataset.DatasetClient.update_information`
+   :ref:`basic_concepts:Segment`               :meth:`~tensorbay.client.dataset.DatasetClient.list_segments`
+                                               :meth:`~tensorbay.client.dataset.DatasetClient.get_segment_object`
+                                               :meth:`~tensorbay.client.dataset.DatasetClient.delete_segments`
+   :ref:`basic_concepts:Catalog & SubCatalog`  :meth:`~tensorbay.client.dataset.DatasetClient.get_catalog`
+                                               :meth:`~tensorbay.client.dataset.DatasetClient.upload_catalog`
+   :ref:`basic_concepts:Dataset`               :meth:`~tensorbay.client.dataset.DatasetClient.commit`
+                                               :meth:`~tensorbay.client.dataset.DatasetClient.update_information`
    ==========================================  ================================================================
 
-Note that :ref:`basic_concepts:Dataset` and :ref:`features/tensorbay_client:Dataset Client`
-are different concepts.
-:class:`Dataset` is a local concept. It represents a dataset entity created locally.
-While :class:`DatasetClient` is a remote concept.
+Note that **dataset** and **dataset client** are different concepts.
+**Dataset** (:ref:`ref <basic_concepts:dataset>`) is a local concept.
+It represents a dataset entity created locally.
+While **dataset client** is a remote concept.
```

Comment:
`**dataset** is a local concept, **daatset client** is a remote concept` is unreasonable.